src/multigrid_iter.cpp

```diff
@@ -181,7 +181,7 @@ void gmgpolar::multigrid_iter()
             double error_difference = fabs(nrm_2_err[it] - nrm_2_err[it - 1]);
             convergence_criterium   = error_difference / nrm_2_err[0];
 
-            if (gyro::icntl[Param::verbose] > 0)
+            if (gyro::icntl[Param::verbose] > 1)
                 std::cout << "--> Iteration " << it << ": error norm = " << nrm_2_err[it]
                           << ", relative error = " << convergence_criterium << std::endl;
         }
```

Comment:
before Line 192 (below) shoudnt we set gyro::icntl[Param::verbose]>0 ? the if-statement is commented out